src/main/java/com/ericsson/ei/jsonmerge/MergeHandler.java

```diff
@@ -77,7 +77,7 @@ public String mergeObject(String id, String mergeId, RulesObject rules, String e
                 preparedToMergeObject = objectToMerge.toString();
             }
 
-            System.out.println("PREPARED OBJECT: " + preparedToMergeObject);
+            // System.out.println("PREPARED OBJECT: " + preparedToMergeObject);
```

Comment:
Good to comment out these System.out but best to remove them.

Reply:
Remove them if its not used, or make a Log debug message if it is good info for debugging.

Reply:
Done.